docs/tutorials/tutorial-0.rst

```diff
@@ -55,7 +55,7 @@ This classfile can run on any Java 6 (or higher) VM. To run the project, type:
 
 .. code-block:: bash
 
-    $ java -classpath ../voc/dist/python-java.jar:. python.example.__init__
+    $ java -classpath ../dist/python-java.jar:. python.example.__init__
```

Comment:
The problem here isn't the classpath - it's the directory in which "mkdir tutorial0" is run. The instructions assume people would go back to the home directory (the directory with `voc` and `env` in it), which is an oversight on my part. The tutorial code shouldn't be dropped in the VOC install directory.

The better fix here is to a "cd .." to the end of the getting started instructions after `ant java`, and a note that this is the "tutorial home directory". Tutorial 0 should then start "If you're not already in the tutorial home directory (the same directory where `voc` and `env` are located), change into that directory".